<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolbox">
     <property name="styleSheet">
      <string notr="true">#toolbox {
	margin: 0;
    padding: 0;
}

::tab:selected {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #6a6ea9, stop: 1 #888dd9);
	color: rgb(255, 255, 255);
	font-weight: normal;
 }

::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FAFBFE, stop: 1 #DCDEF1);
	font-weight: normal;
 }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="details_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>740</width>
        <height>458</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Book Details</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="title"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Authors:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="authors"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Identifiers:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="ids"/>
         </item>
         <item>
          <widget class="QToolButton" name="add_id">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="remove_id">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="edit_id">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="description">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cover:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="cover">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>sintapa.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shelves_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>740</width>
        <height>458</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Tags</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QListWidget" name="tags"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="add_tag">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="remove_tag">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="files_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>740</width>
        <height>458</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Files</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Files:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="fileList"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QToolButton" name="add_file">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="remove_file">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="guess">
       <property name="text">
        <string>&amp;Guess</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
